Dockerfile.windows

```diff
@@ -1,17 +1,28 @@
-FROM microsoft/dotnet:1.1-sdk-msbuild
+FROM microsoft/dotnet-nightly:2.0-sdk-nanoserver as builder
 
 SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
 
+ENV NUGET_XMLDOC_MODE skip
```

Comment:
Can you remove these lines? NUGET_XMLDOC_MODE is already defined in the base image, and DOTNET_SKIP_FIRST_TIME_EXPERIENCE shouldn't apply as the base image has already executed the first-time warmup.